fastlane/Fastfile

```diff
@@ -378,14 +378,15 @@ desc "Update the actions.md on https://docs.fastlane.tools"
 desc "This will also automatically submit a pull request to fastlane/docs"
 lane :update_docs do |options|
   debug = options[:debug]
+  version = options[:version]
 
   unless debug
     verify_env_variables
     ensure_actions_config_items_formatting
   end
 
   template_path = File.expand_path("./assets/render_plugin.md.erb")
-  clone_docs(debug: debug) do
+  clone_docs(debug: debug, version: version) do
```

Comment:
This shouldn't be needed? `clone_docs` doesn't have `version` in its signature 🤔 
```suggestion
  clone_docs(debug: debug) do
```

Reply:
true, not sure where this is coming from.
brain fart.